setup.go

```diff
@@ -130,9 +130,11 @@ func setup(root *cobra.Command) {
 	var cp commands.ConfigFactory
 	var rootDir string
 	var appTag string
+	var forceContext string
 
 	vmConfigFn := vm.ConfigFromCommandParams(root, "vm:", true)
 	cfg := remote.NewConfig(root, "k8s:")
+	root.PersistentFlags().StringVar(&forceContext, "force:k8s-context", "__current__", "Force K8s context")
```

Comment:
default should `""` - we want to use the env's derived context by default

Reply:
DONE